manifest.json

```diff
@@ -31,5 +31,14 @@
     "default_icon": "img/16.png",
     "default_title": "Outline",
     "default_popup": "popup.html"
+  },
+  "commands": {
+    "_execute_page_action": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+E",
```

Comment:
What made you chose this particular shortcut? Why a different default and windows/mac shortcut?